src/sparse/KokkosSparse_gauss_seidel_handle.hpp

```diff
@@ -247,12 +247,12 @@ class GaussSeidelHandle{
   void set_old_to_new_map(const nnz_lno_persistent_work_view_t &old_to_new_map_) {
     this->old_to_new_map = old_to_new_map_;
   }
-  void set_permuted_diagonals (const scalar_persistent_work_view_t permuted_diagonals_){
-    this->permuted_diagonals = permuted_diagonals_;
+  void set_permuted_inverse_diagonal (const scalar_persistent_work_view_t permuted_inverse_diagonal_){
```

Comment:
Does Ifpack2 already call set_permuted_diagonals ?

Reply:
No, it doesn't. This method only sets the permuted diagonal, it does not do any permutations.